### PR TITLE
config: relax the bounds of online configurable worker number (#11651)

### LINKED ISSUE #11776: Better bounding restriction for configuring rocksdb background workers
### What version of TiKV are you using?
/ # ./tikv-server -V
TiKV
Release Version: 5.4.0-alpha
Edition: Community
Git Commit Hash: 99b3436
Git Commit Branch: heads/refs/tags/v5.4.0-nightly
UTC Build Time: 2022-01-04 01:15:55
Rust Version: rustc 1.56.0-nightly (2faabf579 2021-07-27)
Enable Features: jemalloc mem-profiling portable sse test-engines-rocksdb cloud-aws cloud-gcp cloud-azure
Profile: dist_release

### What operating system and CPU are you using?
8core 16G

### Steps to reproduce
modify rocksdb.max-background-jobs from default 7 to 8

### What did you expect?
can modify rocksdb.max-background-jobs success within a reasonable range

### What did happened?
modify rocksdb.max-background-jobs from default 7 to 8， modify failed

### PR DESCRIPTION
**Pull request by ethercflow**
close #11776

Signed-off-by: Wenbo Zhang <ethercflow@gmail.com>

### What is changed and how it works?

Close #11776  

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- None

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/store/config.rs [hunk 1]
```diff
@@ -495,11 +495,11 @@ impl Config {
         // Since the following configuration supports online update, in order to
         // prevent mistakenly inputting too large values, the max limit is made
         // according to the cpu quota.
```
Comment (5kbpers) [inline components/raftstore/src/store/config.rs:497]: Could you clarify the factor `10` in the comment?

Comment (ethercflow) [reply components/raftstore/src/store/config.rs]: Done

**Review by gengliqi — APPROVED** [review]

**Review by 5kbpers — APPROVED** [review]

**Comment by ethercflow** [comment]
@tabokie shrink restriction has been removed, PTAL

**Review by tabokie — APPROVED** [review]

**Pull request merged**